packages/taiko-client/prover/init.go

```diff
@@ -98,9 +98,10 @@ func (p *Prover) initProofSubmitters(
 ) error {
 	for _, tier := range p.sharedState.GetTiers() {
 		var (
-			producer  proofProducer.ProofProducer
-			submitter proofSubmitter.Submitter
-			err       error
+			bufferSize = p.cfg.SGXProofBufferSize
```

Comment:
Shall we always init `bufferSize` here to `0` at first?

Reply:
If there is no op, it is only used by `SGXProofProducer`.
But in devnet and current hekla(for user), we have op and sgx, it will be used by `SGXProofProducer` and `OptimisticProofProducer`.